examples/nodejs/stock.js

```diff
@@ -2,29 +2,36 @@
 
 var http = require('http');
 
-module.exports = function (context, callback) {    
+module.exports = async function (context) {
     let body = context.request.body;
     console.log(`body text: ${body['text']}`);
 
     var symbol = body['text'].split(' ')[1];
 
-    http.get({
-        host: 'finance.google.com',
-        path: `/finance/info?q=NYSE:${symbol}`
-    }, function(response) {
-        var resp = '';
-        response.on('data', function(d) {
-            resp += d;
-        });
-        response.on('end', function() {
-
-            try {
-                var parsed = JSON.parse(resp.slice(3));
-                var lastTrade = parsed[0]['l_cur']
-                callback(200, `{ "text": "${symbol} last traded at ${lastTrade}" }`);
-            } catch (e) {
-                callback(200, `{ "text": "Error (invalid NYSE symbol?)" }`);
-            }
+    return new Promise(function(resolve, reject) {
+        http.get({
```

Comment:
I don't mind using a dependency if it is very widely used and makes the code easier to read and write. (But we can totally do this later, I don't want to block this PR for this example.)

In your PR description above, you seem to indicate you can use `await request` without adding a dependency; maybe I misunderstood it?